PurchasesCoreSwift/Logging/Strings/AttributionStrings.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  AttributionStrings.swift
+//  PurchasesCoreSwift
+//
+//  Created by Andrés Boedo on 9/14/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCAttributionStrings) public class AttributionStrings: NSObject {
+    @objc public var instance_configured_posting_attribution: String { "There is an instance configured, posting attribution." }
+    @objc public var no_instance_configured_caching_attribution: String { "There is no instance configured, caching attribution." }
+}
```

Comment:
I really like to have one file of strings per module/scope. I am not sure if @tiinanguyen intention was to have a single file though, to be able to search/see through it easily. I don't think it's too much of a problem to have them split in separate files and I think the benefits outcome the drawbacks.

Reply:
from experience, I'd highly recommend splitting into multiple files, otherwise it gets pretty unwieldy and it's far more likely to cause merge conflicts if more than one developer is working on the project at a time. 
in the past I've even had these alphabetized within each file, so as to minimize merge conflicts

Reply:
Agreed, in the proposed design as the number of messages grow, it will definitely become a large chunk of code in the future. Splitting the files by group seems have more benefits and avoiding potential merge conflicts 👍

Reply:
Any reason why these are variables instead of constants?

Reply:
forgot to reply here - I went with variables so that they wouldn't be loaded into memory unnecessarily

Reply:
these aren't really variables in that they're not stored, they're computed properties, so they work like functions. 
under this mechanism, the AttributionStrings object should only use the minimal amount of memory possible for an NSObject